### PR TITLE
OpenSCAP json tailoring (HMS-4462)

### PR DESCRIPTION
**Pull request by kingsleyzissou**
This depends on https://github.com/osbuild/osbuild-composer/pull/4272

### CODE REVIEW

File: docs/user-guide/01-blueprint-reference.md [hunk 1]
```diff
@@ -1242,6 +1243,46 @@ unselected = [ "grub2_password" ]
 </TabItem>
 </Tabs>
 
+### OpenSCAP JSON Tailoring
+
+The OpenSCAP team has developed a [json schema](https://github.com/ComplianceAsCode/schemas/blob/b91c8e196a8cc515e0cc7f10b2c5a02b4179c0e5/tailoring/schema.json)
+as an abstraction to the standard XML tailoring file. This approach helps simplify the process of tailoring an OpenSCAP profile.
+A basic example can be found [here](https://github.com/ComplianceAsCode/schemas/blob/b91c8e196a8cc515e0cc7f10b2c5a02b4179c0e5/tailoring/example-basic.json_tailoring).
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: link is dead :-/
the `_tailoring` seems to be too much
```suggestion
A basic example can be found [here](https://github.com/ComplianceAsCode/schemas/blob/b91c8e196a8cc515e0cc7f10b2c5a02b4179c0e5/tailoring/example-basic.json).
```

File: docs/user-guide/01-blueprint-reference.md [hunk 2]
```diff
@@ -1242,6 +1243,46 @@ unselected = [ "grub2_password" ]
...
+
+Both the `profile_id` and the `filepath` options are required and the `profile_id` needs to match the `profile_id` used in the json tailoring file.
+
+Note: users will need to use [custom files](#Files) to save the json tailoring file to the image. The generated xml tailoring file is saved to
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: I think this has to be written in small letters to work always
```suggestion
Note: users will need to use [custom files](#files) to save the json tailoring file to the image. The generated xml tailoring file is saved to
```

**Review by schuellerf — CHANGES_REQUESTED** [review]

**Comment by schuellerf** [comment]
sorry - did a rebase - I hope that was ok :-|

File: docs/user-guide/01-blueprint-reference.md (hunk 1 above)
Comment (kingsleyzissou) [reply docs/user-guide/01-blueprint-reference.md]: This was a silly typing mistake. Thanks for finding

**Comment by kingsleyzissou** [comment]
> sorry - did a rebase - I hope that was ok :-|

No worries, I've just fixed and rebased again

File: docs/user-guide/01-blueprint-reference.md [hunk 3]
```diff
@@ -1242,6 +1243,46 @@ unselected = [ "grub2_password" ]
 </TabItem>
 </Tabs>
 
+### OpenSCAP JSON Tailoring
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: please add this bullet for the overview (and then also the manual anchor is needed)
```suggestion
### OpenSCAP JSON Tailoring 🔵 {#openscap-json-tailoring}
```

File: docs/user-guide/01-blueprint-reference.md [hunk 4]
```diff
@@ -1242,6 +1243,46 @@ unselected = [ "grub2_password" ]
...
+
+The OpenSCAP team has developed a [json schema](https://github.com/ComplianceAsCode/schemas/blob/b91c8e196a8cc515e0cc7f10b2c5a02b4179c0e5/tailoring/schema.json)
+as an abstraction to the standard XML tailoring file. This approach helps simplify the process of tailoring an OpenSCAP profile.
+A basic example can be found [here](https://github.com/ComplianceAsCode/schemas/blob/b91c8e196a8cc515e0cc7f10b2c5a02b4179c0e5/tailoring/example-basic.json).
+
+Both the `profile_id` and the `filepath` options are required and the `profile_id` needs to match the `profile_id` used in the json tailoring file.
+
+Note: users will need to use [custom files](#files) to save the json tailoring file to the image. The generated xml tailoring file is saved to
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: nit-pick
```suggestion
Note: users will need to use [custom files](#files) to save the json tailoring file to the image. The generated XML tailoring file is saved to
```

**Review by schuellerf — CHANGES_REQUESTED** [review]

**Review by ochosi — APPROVED** [review]
LGTM!

**Review by schuellerf — APPROVED** [review]

**Pull request merged**